src/pages/verify/guides/custom-styling.mdx

```diff
@@ -11,11 +11,18 @@ import customStyling from './images/custom-styling.png';
 export default Layout;
 
 Some of the eID methods require additional dialogs outside of those of the actual eID service.
-These may be styled to you liking using CSS as described below.
+These may be styled and enhanced to you liking using one of the following approaches: 
 
-## Preview of the user-facing dialogs
+1. [Adding custom CSS](#custom-css), if your goal is to add basic styling
+2. [Adding custom HTML](#custom-html) for creating more complex UIs (e.g. a layout with extra information around the Criipto login box).
 
-You can see a preview of the dialogs in the `Styling` tab for your tenant in the management dashboard on [dashboard.criipto.com](https://dashboard.criipto.com/styling) - such as the ones for SE BankID:
+## Custom CSS
```

Comment:
Maybe we should pick another word than "custom CSS" to differentiate it more from "custom HTML and CSS".

Reply:
"Adding a stylesheet" maybe

Reply:
or "Adding your own stylesheet"